spec/lib/oauth/client/credentials_spec.rb

```diff
@@ -29,16 +29,16 @@ class Doorkeeper::OAuth::Client
       it 'stops at the first credentials found' do
         not_called_method = double
         expect(not_called_method).not_to receive(:call)
-        credentials = Credentials.from_request request, ->(r) {}, method, not_called_method
+        Credentials.from_request request, ->(r) {}, method, not_called_method
```

Comment:
Unused block argument - `r`. If it's necessary, use `_` or `_r` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.